apps/studio/pages/project/[ref]/reports/api-overview.tsx

```diff
@@ -108,7 +110,12 @@ export const ApiReport: NextPageWithLayout = () => {
 
 // hook to fetch data
 const useApiReport = () => {
+  const { project } = useProjectContext()
   const { ref: projectRef } = useParams()
+  const state = useDatabaseSelectorStateSnapshot()
+
+  // [Joshen] TODO: Once API support is out
+  const showReadReplicasUI = false // project?.is_read_replicas_enabled
```

Comment:
Isn't `project?.is_read_replicas_enabled` a thing? 🤨

Reply:
^ the API support for read replicas on API reports is not out yet 😬